app/styles/_variables.scss

```diff
@@ -5,3 +5,18 @@ $reviews-title-hover: #D5E6ED;
 $gradient-percentage: 15%;
 $nav-bar-height: 25px;
 $carousel-control-width: 10%;
+
+$color-action-submit: #CABC14;
+$color-action-accept: #5CB85C;
+$color-action-reject: #D9534F;
+$color-action-edit-comment: #337ab7;
+
+$color-text-black: #151515;
```

Comment:
Yep, for the bold/large title of each review log on the dashboard, e.g. "Karl Marx accepted a preprint". I'll redefine black to #333333 and use it more consistently.